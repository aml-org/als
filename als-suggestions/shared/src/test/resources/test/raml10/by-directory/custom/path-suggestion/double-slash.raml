#%RAML 1.0

traits:
  client_id_enforcement: !include //*