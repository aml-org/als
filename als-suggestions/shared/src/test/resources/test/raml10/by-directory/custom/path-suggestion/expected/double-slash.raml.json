[
  {
    "label": "api-trait-include.raml",
    "kind": [
      
    ],
    "detail": "unknown",
    "documentation": "Path suggestion",
    "deprecated": [
      
    ],
    "preselect": [
      
    ],
    "sortText": "11api-trait-include.raml",
    "filterText": "//api-trait-include.raml",
    "insertText": null,
    "insertTextFormat": [
      1
    ],
    "textEdit": {
      "range": {
        "start": {
          "line": 3,
          "character": 34
        },
        "end": {
          "line": 3,
          "character": 36
        }
      },
      "newText": "//api-trait-include.raml"
    },
    "additionalTextEdits": null,
    "commitCharacters": null,
    "command": null
  },
  {
    "label": "api-trait-inlined.raml",
    "kind": [
      
    ],
    "detail": "unknown",
    "documentation": "Path suggestion",
    "deprecated": [
      
    ],
    "preselect": [
      
    ],
    "sortText": "11api-trait-inlined.raml",
    "filterText": "//api-trait-inlined.raml",
    "insertText": null,
    "insertTextFormat": [
      1
    ],
    "textEdit": {
      "range": {
        "start": {
          "line": 3,
          "character": 34
        },
        "end": {
          "line": 3,
          "character": 36
        }
      },
      "newText": "//api-trait-inlined.raml"
    },
    "additionalTextEdits": null,
    "commitCharacters": null,
    "command": null
  },
  {
    "label": "api.raml",
    "kind": [
      
    ],
    "detail": "unknown",
    "documentation": "Path suggestion",
    "deprecated": [
      
    ],
    "preselect": [
      
    ],
    "sortText": "11api.raml",
    "filterText": "//api.raml",
    "insertText": null,
    "insertTextFormat": [
      1
    ],
    "textEdit": {
      "range": {
        "start": {
          "line": 3,
          "character": 34
        },
        "end": {
          "line": 3,
          "character": 36
        }
      },
      "newText": "//api.raml"
    },
    "additionalTextEdits": null,
    "commitCharacters": null,
    "command": null
  },
  {
    "label": "double-slash.raml",
    "kind": [
      
    ],
    "detail": "unknown",
    "documentation": "Path suggestion",
    "deprecated": [
      
    ],
    "preselect": [
      
    ],
    "sortText": "11double-slash.raml",
    "filterText": "//double-slash.raml",
    "insertText": null,
    "insertTextFormat": [
      1
    ],
    "textEdit": {
      "range": {
        "start": {
          "line": 3,
          "character": 34
        },
        "end": {
          "line": 3,
          "character": 36
        }
      },
      "newText": "//double-slash.raml"
    },
    "additionalTextEdits": null,
    "commitCharacters": null,
    "command": null
  },
  {
    "label": "expected/",
    "kind": [
      
    ],
    "detail": "unknown",
    "documentation": "Path suggestion",
    "deprecated": [
      
    ],
    "preselect": [
      
    ],
    "sortText": "11expected/",
    "filterText": "//expected/",
    "insertText": null,
    "insertTextFormat": [
      1
    ],
    "textEdit": {
      "range": {
        "start": {
          "line": 3,
          "character": 34
        },
        "end": {
          "line": 3,
          "character": 36
        }
      },
      "newText": "//expected/"
    },
    "additionalTextEdits": null,
    "commitCharacters": null,
    "command": null
  }
]